app/src/main/res/layout/fragment_detail.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        xmlns:tools="http://schemas.android.com/tools"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/detailLayout">
+
+    <TextView
+            android:text="@string/create_book_item"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/detail_frag_header"
+            android:textSize="30sp"
+            tools:layout_conversion_absoluteHeight="0dp"
```

Comment:
tools:layout_conversion_absoluteHeight="0dp" a ten pod nim odstranit... tools znamena ze stejne neni videt, je to pomocny nastroj